### ISSUE TITLE
VoiceOver reads back all options prior to interaction 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-input#185]
## Describe the bug

When using a screen reader to access content within the `auro-select` element, the screen reader reads back all the options in the menu. This is not only distracting, but this will bring things to a stop if there is a long list of options. 

## To Reproduce

1. Go to https://auro.alaskaair.com/components/auro/select
1. Activate screen reader
1. Interact with auro-select element
1. Hear error

![ca4de323-4e0f-4ea6-a68c-8ffdb407d38c](https://user-images.githubusercontent.com/181089/191128952-540f1033-7760-42f5-84bc-fbd3125db56c.jpg)

Additionally, when in the select option, the screen reader does not read back the label of the element. This is a crucial miss. 

## Expected behavior

Please see https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select and mirror experience.
